.goreleaser.yml

```diff
@@ -50,11 +50,14 @@ brews:
     owner: mongodb
     name: homebrew-brew
   folder: Formula
-  skip_upload: false
+  skip_upload: auto
```

Comment:
Just checking, so this means it will skip uploading rc's but will upload the tags?

Reply:
Rcs, betas, alpha, and such, we disabled this when we were doing our rc, so going back to not publishing unstable